### DISCUSSION TITLE
视频详情页点选下方推荐视频后焦点归位问题改进建议

### DISCUSSION THREAD

**Discussion by CaptainBott** [Paladinfeng/MiaoProject#195] [Ideas]
### 使用环境
Apple TV 4K (第一代) A1842、Siri Remote、tvOS 16.4.1、Miao Project version 2.1.3 build 102
### App 状态
当前页面：视频详情页
页面底部“推荐视频”列表状态：已正常加载完毕
焦点位置：“推荐视频”列表任意列表项
### 用户动作
使用 Siri Reomte 确认键按下“推荐视频”列表中的某一列表项
### 当前版本功能
视频详情页切换至点选的视频、焦点移动至“播放”按钮
### 遇到的问题
因 Siri Remote 的五向键带有触控功能，当使用拇指按下确认键后极短时间内（此期间因焦点已移动至“播放”按钮，屏幕开始向上卷动，且未完成卷动），触控区域会识别拇指在触控区域内的极细微移动，有相当大的几率，向系统发送上下左右某一或某几个的移动指令。带屏幕卷动动作完成后，焦点将会从“播放”按钮移动到临近的其他按钮。
### 改进建议
能否在焦点移动至“播放”按钮后的屏幕卷动时间内（约0.3~0.5s），不允许用户触发焦点移动（不允许用户使用方向键或不接收来自触控板的方向指令），待焦点移动+屏幕卷动完成后，再允许用户触发焦点移动（允许用户使用方向键或接收来自触控板的方向指令）。又或者屏幕卷动期间直接不接收来自遥控器的所有指令，待屏幕卷动完成后再响应遥控器指令。
### 其他
以上说的这种焦点因“误触”而不在位于“播放”按钮上的情况经常会造成我的很多困扰，此时我的目的也正是想要按下“播放”按钮，而非其他按钮。希望作者能考虑下这个提议，再次向为了这个项目默默付出了很多心血的作者表示感谢。

**Comment by Paladinfeng** [comment]
刷新的问题将在下个版本修复，不过我最近住院了，更新要稍稍延期

**Reply by CaptainBott** [reply]
非常感谢！祝早日痊愈。
